lib/net/make_net_client.ts

```diff
@@ -26,11 +26,11 @@ export const makeNetClient = (cfg: Net.ClientConfig, logger?: Logger): Net.Clien
           url: _url.toString(),
           headers: {
             ...cfg.headers,
-            ...opts.headers,
+            ...opts?.headers,
           },
-          method: opts.method,
-          data: opts.body,
-          params: opts.params,
+          method: opts?.method,
+          data: opts?.body,
+          params: opts?.params,
```

Comment:
my previous comment here doesn't seem to be resolved

Reply:
Do not know why this commit did not appear . Pushing it in new PR